### ISSUE TITLE
Tint colour not respected

### ISSUE THREAD

**Issue by sumowesley** [xmartlabs/XLForm#723]
I have a couple of buttons on one of my forma and they steadfastly refuse to adopt the global tint colour for the app. I've tried a basic tint colour (set in appdelegate) inside the example XLForm app and can see similar behaviour:
![xlform_tint_160206a](https://cloud.githubusercontent.com/assets/5694619/12865677/fa52e7e4-ccaa-11e5-963a-f11ed898841b.png)
Am I right in thinking that the item show should be red and not blue?

**Comment by mats-claassen** [comment]
Hi, XLForm is currently [not setting the textLabel color to the tintColor](https://github.com/xmartlabs/XLForm/blob/master/XLForm/XL/Cell/XLFormBaseCell.m#L68). You have to configure the cells yourself.

**Comment by sumowesley** [comment]
Hi Matt, this wasn't limited to textField - I think the image above is a selector field and I've noticed it on button fields. I has to be picking up the blue colour from somewhere because at the time the form is being rendered, the global tint has been set.

Curiously - the segment control and the booleanSwitch both honour the globally set tint colour. If I apply the globally set tint colour as part of the form setup (in addition to setting it globally in the appdelegate) I seem to get the behaviour I am expecting e.g. the button I have configured is no longer blue text and the active textField being editing does not have the blue colour for the label.
